fossunited/api/emailing.py

```diff
@@ -0,0 +1,451 @@
+from typing import Literal
+
+import frappe
+
+from fossunited.api.chapter import check_if_chapter_member
+from fossunited.doctype_ids import EVENT
+
+EMAIL_GROUP_TYPES = Literal[
+    "Chapter Master",
```

Comment:
nit - I would prefer not using the Master/Slave terminology. Can we go with `Chapter main`?

And does `Chapter main` mean all people who have ever attended a chapter meetup?

Reply:
1. will change
2. Yes